src/app/components/Item/SearchResultsItems.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import ItemTable from './ItemTable'
+import ItemsDefinitionLists from './ItemsDefinitionLists'
+
+import { MediaContext } from '../Application/Application'
+
+const SearchResultsItems = ({ items, bibId, id, searchKeywords, page }) => {
+
+  return (
+    <MediaContext.Consumer>
```

Comment:
there's a simpler syntax for functional components. you can use `const {media} = useContext(MediaContext)` instead. though i guess not super necessary.